### PR TITLE
Added Coupling & Cohesion examples

### PR DESCRIPTION
**Pull request by kharaei**
I tried to implement simple and understandable examples.

### CODE REVIEW

File: sources/Reza Kharaei/Design.Cohesion/TemporalCohesion.cs
Comment (mohammadKarimi) [inline sources/Reza Kharaei/Design.Cohesion/TemporalCohesion.cs:1]: it isn't Temporal cohesion, it is just like Procedural

File: sources/Reza Kharaei/Design.Coupling/CommonCoupling.cs
Comment (mohammadKarimi) [inline sources/Reza Kharaei/Design.Coupling/CommonCoupling.cs:1]: Where is Common composite data structure?

File: sources/Reza Kharaei/Design.Coupling/StampCoupling.cs
Comment (mohammadKarimi) [inline sources/Reza Kharaei/Design.Coupling/StampCoupling.cs:1]: I can't see any Stamp coupling here in your code.

**Review by mohammadKarimi — CHANGES_REQUESTED** [review]
there are some issues on your code. please fix them then send for feedback.

File: sources/Reza Kharaei/Design.Coupling/StampCoupling.cs
Comment (kharaei) [reply sources/Reza Kharaei/Design.Coupling/StampCoupling.cs:1]: thank you for comment, I think When calling the SendEmail method in PostAction, there is no need for a mobile number.

**Pull request closed without merging**